### PR TITLE
[wasm] Bump emscripten version.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by kjpou1** [comment]
@monojenkins build failed

**Comment by lewurm** [comment]
can you rewrite the commit history so that the cindex.py changes are visible in the history?

also would probably be worth to upstream them here: https://github.com/ethanhs/clang

File: tools/offsets-tool-py/Makefile [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ACTIVATE=offtool/bin/activate
 
 setup: $(ACTIVATE)
-	. $(ACTIVATE) && pip3 install clang
```
Comment (lewurm) [inline tools/offsets-tool-py/Makefile]: is this a temporary thing? if not, we can remove this `virtualenv` stuff.

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

File: tools/offsets-tool-py/Makefile (hunk 1 above)
Comment (vargaz) [reply tools/offsets-tool-py/Makefile]: Doesn't look like that repo is getting updated.

**Comment by vargaz** [comment]
@monojenkins build failed

File: tools/offsets-tool-py/Makefile (hunk 1 above)
Comment (vargaz) [reply tools/offsets-tool-py/Makefile]: Will clean up in a follow-up PR.

**Comment by vargaz** [comment]
Does adding these llvm sources cause any licensing issues ?

**Pull request merged**